test/parallel/test-process-argv1.mjs

```diff
@@ -0,0 +1,25 @@
+import { spawnPromisified } from '../common/index.mjs';
+import * as fixtures from '../common/fixtures.mjs';
+import { strictEqual } from 'node:assert';
+import { describe, it } from 'node:test';
+import { dirname } from 'node:path';
+
+describe('process.argv1', { concurrency: true }, () => {
+  it('should be the string for the entry point before the entry was resolved into an absolute path', async () => {
+    const fixtureRelativePath = './process-argv1.js';
+    const fixtureAbsolutePath = fixtures.path(fixtureRelativePath);
+
+    const { code, signal, stdout, stderr } = await spawnPromisified(process.execPath, [fixtureRelativePath], {
+      cwd: dirname(fixtureAbsolutePath),
```

Comment:
Can we run this test without `node:test` and `spawnPromisified`? If all we are doing is validating `process.argv1` and `process.argv[1]`, why not do a simple assertion?

```
assert.strictEqual(process.argv1, __filename)
```

Reply:
`__filename` would be the resolved absolute path. `process.argv1` is the pre-resolution path.